TidepoolOnboarding/View Controllers/PrescriptionReviewUICoordinator.swift

```diff
@@ -105,8 +111,10 @@ class PrescriptionReviewUICoordinator: UINavigationController, CompletionNotifyi
                 self?.setupCanceled()
             }
             prescriptionViewModel.didFinishStep = { [weak self] in
-                self?.therapySettingsViewModel = self?.constructTherapySettingsViewModel()
-                self?.stepFinished()
+                guard let self = self else { return }
+                self.prescriptionReviewDelegate?.prescriptionReview(hasNewPrescription: self.prescriptionViewModel.prescription!)
```

Comment:
why force unwrap here?  Is it not possible to pass in an optional to the function and have it decide what to do? Or should `self.prescriptionViewModel.prescription` be part of the `guard`?

Reply:
A prescription is 100% required by this point. If there is no prescription, then it is a complete logic fail and the app will not function correctly. I can add this to the guard statement, but if prescription does happen in real life, then they user will be stuck at a screen, tapping a button that does nothing.

Also, much of this has to be refactored a bit to better fit into onboarding. So, I'm not going to make the change at this point, but will consider this case when refactoring to hopefully prevent the forced unwrap in the first place.